cwms-data-api/src/test/java/cwms/cda/data/dao/StreamLocationDaoTestIT.java

```diff
@@ -0,0 +1,260 @@
+import cwms.cda.helpers.DTOMatch;
+import fixtures.CwmsDataApiSetupCallback;
+import fixtures.TestAccounts;
+import java.io.IOException;
+import java.sql.SQLException;
+import java.time.ZoneId;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import mil.army.usace.hec.test.database.CwmsDatabaseContainer;
+import org.jooq.DSLContext;
+import org.jooq.exception.DataAccessException;
+import org.junit.jupiter.api.AfterAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+@Tag("integration")
+final class StreamLocationDaoTestIT extends DataApiTestIT {
+
+    private static final String OFFICE_ID = TestAccounts.KeyUser.SWT_NORMAL.getOperatingOffice();
+    private static final List<Location> LOCATIONS_CREATED = new ArrayList<>();
+    private static final List<String> STREAM_LOC_IDS = new ArrayList<>();
+    private static final List<Stream> STREAMS_CREATED = new ArrayList<>();
+
+    @BeforeAll
+    public static void setup() {
+        try {
+            for (int i = 0; i < 2; i++) {
+                String testLoc = "STREAM_LOC_" + System.currentTimeMillis()/10;
+                STREAM_LOC_IDS.add(testLoc);
+                createAndStoreTestLocation(testLoc, "STREAM_LOCATION");
+                createAndStoreTestStream(testLoc + "S");
+            }
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    private static void createAndStoreTestStream(String testLoc) throws SQLException {
+        createAndStoreTestLocation(testLoc, "STREAM");
+        CwmsDatabaseContainer<?> db = CwmsDataApiSetupCallback.getDatabaseLink();
+        db.connection(c-> {
+            StreamDao streamDao = new StreamDao(getDslContext(c, OFFICE_ID));
+            Stream streamToStore = new Stream.Builder()
+                    .withId(new CwmsId.Builder()
+                            .withOfficeId(OFFICE_ID)
+                            .withName(testLoc)
+                            .build())
+                    .withLength(100.0)
+                    .withLengthUnits("km")
+                    .build();
+            STREAMS_CREATED.add(streamToStore);
+            streamDao.storeStream(streamToStore, true);
+        });
+    }
+
+    @AfterAll
+    public static void tearDown() {
+        for(Location location : LOCATIONS_CREATED){
+            try {
+                CwmsDatabaseContainer<?> db = CwmsDataApiSetupCallback.getDatabaseLink();
+                db.connection(c-> {
+                    LocationsDao locationsDao = new LocationsDaoImpl(getDslContext(c, OFFICE_ID));
+                    locationsDao.deleteLocation(location.getName(), location.getOfficeId(), true);
+                });
```

Comment:
pass webuser from CwmsDataApiSetupCallback as parameter to db.connection

Reply:
passed in webUser param